PSIni/Functions/Get-IniContent.ps1

```diff
@@ -160,6 +164,18 @@
             }
         }
         Write-Verbose "$($MyInvocation.MyCommand.Name):: Finished Processing file: $FilePath"
+        if($IgnoreEmptySections){
+            $ToRemove = [System.Collections.ArrayList]@()
+            foreach($Section in $ini.Keys){
+                if(($ini[$Section]).Count -eq 0){
+                    $ToRemove.Add($Section)
```

Comment:
this line produces console output with the index of the array where the item was added to (see screenshot).
prefix this line with `$null = ` to capture (and hide) this write to stdout

<img width="613" alt="image" src="https://github.com/lipkau/PsIni/assets/4992870/6fadc4f2-d90e-4daf-9ff9-77d9f9ff1bb5">